### ISSUE TITLE
Add message event for private unsecure loan sweden. 

### ISSUE THREAD

**Issue by Torbacka** [open-broker/specification#89]
There is more and more banks that need a message event. It need to be able to send message both ways.

**Comment by mantono** [comment]
Ah, so even from broker to bank as well? Are there many banks that supports it?

**Comment by Torbacka** [comment]
Yeah there is at least two banks that would want that. The alternative is that we write a lot of emails.

**Comment by mantono** [comment]
It seems like a slightly different use case. I imagine Message more as informational messages in one direction rather than a dialog between two parties (were most messages are machine generated). What would it give you to send a response to a machine generated message? Is it not possible that during such circumstances that email is actually better? For example, if only two banks support sending messages to, what will happen when you send a message to another bank? Will the message be sent to the integration (which must support it?) which then quietly drops it because it has no where to send it?

Sorry if I sound harsh, I just think we need to consider all "ifs" and "buts" :wink:
